src/main/frontend/dicts.cljc

```diff
@@ -3730,8 +3730,335 @@
 
         :file-sync/other-user-graph "Geçerli yerel grafik, diğer kullanıcının uzak grafiğine bağlıdır. Bu yüzden senkronizasyon başlatılamıyor."
         :file-sync/graph-deleted "Geçerli uzak grafik silindi"}
-
-   :tongue/fallback :en})
+   
+   :ko {:tutorial/text #?(:cljs (rc/inline "tutorial-ko.md")
```

Comment:
Looks like tests are failing because this file doesn't exist. You can either translate the tutorial or just refer to `tutorial-en.md`